src/OpenApi/src/Microsoft.AspNetCore.OpenApi.csproj

```diff
@@ -7,6 +7,10 @@
     <!-- Needed to support compiling Utf8BufferTextWriter implementation shared with SignalR -->
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Description>Provides APIs for annotating route handler endpoints in ASP.NET Core with OpenAPI annotations.</Description>
+    <IsAotCompatible>true</IsAotCompatible>
+    <!-- Required to generated native AoT friendly `MapOpenApi` endpoint. -->
+    <EnableRequestDelegateGenerator>true</EnableRequestDelegateGenerator>
+    <InterceptorsPreviewNamespaces>$(InterceptorsPreviewNamespaces);Microsoft.AspNetCore.Http.Generated</InterceptorsPreviewNamespaces>
```

Comment:
```suggestion
```

Can this be removed? I thought it was done automatically.

Reply:
It's only done automatically for projects that use the Web SDK.